vuu-ui/showcase/src/examples/Layout/SavePanel.examples.tsx

```diff
@@ -14,7 +14,7 @@ export const SavePanel = () => {
       isOpen
       style={{ maxHeight: 500, borderColor: "#6d188b" }}
       title={"Save Layout"}
-      hideCloseButton
+      hideCloseButton={true}
```

Comment:
`hideCloseButton={true}` is functionally equivalent to `hideCloseButton`. When a component prop is boolean and we want to pass true to it we can just have the property name without the `={true} ` see isOpen prop on the same component

Reply:
[2a2dd2f](https://github.com/ScottLogic/finos-vuu/pull/100/commits/2a2dd2ff66595bf9dc891d45a9d49409c7f12bba)